cli/running/watchdog_test.go

```diff
@@ -4,13 +4,14 @@ import (
 	"io"
```

Comment:
The test shold pass fine on each commit. Please squash it to the previous one.

Reply:
Changed